code/modules/clothing/under/accessories/storage.dm

```diff
@@ -60,7 +60,11 @@
 	desc = "Worn brownish synthcotton vest with lots of pockets to unload your hands."
 	icon_state = "vest_brown"
 	slots = 5
-	
+
+/obj/item/clothing/accessory/storage/brown_vest/explorer
+	name = "explorer's webbing"
```

Comment:
There is already an item with a very similar name and similar purpose, miners can already get webbings via loadout, either just give a regular one in their locker. I see no real reaso nto add a better one to them besides more powercreep.